### ISSUE TITLE
Missing Job Glue Name at GlueTransformStage

### ISSUE THREAD

**Issue by jvjfranca** [awslabs/aws-ddk#181]
### Describe the bug

When instantiating a GlueTransformStage without a predefined crawler/job the deployment of the state machine is failing.

Missing job_name and crawler_name used by the state machine tasks.

### How to Reproduce

        glue_stage = GlueTransformStage(
            self,
            id='xpto-gluestage',
            environment_id=environment_id,
            database_name="xpto-database",
            targets=cdk_glue.CfnCrawler.TargetsProperty(
                s3_targets=[
                    cdk_glue.CfnCrawler.S3TargetProperty(
                        path=f"s3://{xpto_data.bucket_name}/raw/"
                    )
                ]
            ),
            crawler_role=glue_role,
            job_args={
                "--S3_SOURCE_PATH": xpto_data.arn_for_objects("raw/"),
                "--S3_TARGET_PATH": xpto_data.arn_for_objects("stage/"),
            }
        )

### Screenshots

![image](https://user-images.githubusercontent.com/16049492/199265028-221ae8d6-097e-416e-9061-ed2f8fb92847.png)

![image](https://user-images.githubusercontent.com/16049492/199265184-be3c8299-ef18-4d4d-a803-e4fc6642cd23.png)

### Environment

```
Provide your `pip list` output, particularly the version of the AWS DDK library you used. Providing this information may significantly improve resolution times.
```

### OS

Linux

### Python version

3.10

### AWS DDK version

0.5.1

**Comment by malachi-constant** [comment]
We are planning to release a new version shortly but I believed this issue was fixed [here](https://github.com/awslabs/aws-ddk/pull/167). You can also take a look at the [Glue example](https://github.com/aws-samples/aws-ddk-examples/blob/main/simple-glue-transform/app.py) for minimally required args to use the stage.

Before `0.6.0` is available you will need to pip install from `main` 

```
pip install git+https://github.com/awslabs/aws-ddk.git#subdirectory=core
```

**Comment by malachi-constant** [comment]
This is fixed in [`0.6.0`](https://github.com/awslabs/aws-ddk/releases/tag/0.6.0)

**Comment by jvjfranca** [comment]
Thank you!

Em qui., 3 de nov. de 2022 19:03, Lucas Hanson ***@***.***>
escreveu:
